docs/storefront/headless/routes.mdx

```diff
@@ -0,0 +1,269 @@
+# Routes with the GraphQL Storefront API
+
+The BigCommerce GraphQL Storefront API allows you to retrieve data at the destination of a route (URL) for each storefront channel using the `route` node. This option makes it easier to build headless storefront applications, removing the need to track entity ID and type when accessing data for entities such as Products, Categories, Pages, Blogs, and more.
+
+The only required input when using the `route` node is the destination URL. The response will be an entity at that URL, a redirect to a new location, or nothing if the URL does not relate to an entity or redirect.
+
+When querying a `route`, the response may include the following node types:
+
+* Product
+* Category
+* Brand
+* NormalPage
+* ContactPage
+* RawHtmlPage
+* Blog
+* BlogPost
+
+## Redirect support
+
+The `route` node supports 301 Redirects that you have been added to the store. If the URL provided as input to the `route`, references a redirect, then the response will return a `Redirect` node, which comes in two varieties: a manual redirect or a dynamic redirect.
```

Comment:
> The `route` node supports 301 Redirects that you have been added to the store.

Looks like a typo...this should read as:
The `route` node supports 301 Redirects that have been added to the store.
or
The `route` node supports 301 Redirects that you have added to the store.